tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -166,6 +169,19 @@ Verify RHODS Release Version Number
     ${version} =  Get RHODS Version
     Should Match Regexp    ${version}    ^[0-9]+\.[0-9]+\.[0-9]+\(-[0-9]+)*$
 
+Verify That Alert Notification Emails Can Be Updated And Changed In Alertmanager Configuration
+    [Documentation]    Vrifies the Alert Notification email is updated in Addon-Managed-Odh-Parameters Secret and Alertmanager ConfigMap
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-673
+    ...       KnownIssues
+    ${email_to_change} =    Set Variable    dummyemail2@redhat.com
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_name} =    Get Cluster Name By Cluster ID    ${cluster_id}
+    Update Notification Email Address    ${cluster_name}    ${email_to_change}
+    ${email_from_secret} =    Get Notification Email From Addon-Managed-Odh-Parameters Secret
+    Should Be Equal As Strings    ${email_to_change}    ${email_from_secret}
+    Check Email Is Present In Alertmanager ConfigMap    ${email_to_change}
```

Comment:
Even if there is a bug in RHODS here, alertmanager.cfg will probably need a few minutes (<5 minutes) to be updated. I think you should create also a keyword `Wait Until Notification Email From Alertmanager ConfigMap Contains $email $retries `

More info at 
https://issues.redhat.com/browse/RHODS-1974?focusedCommentId=19937175&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19937175